walletsetup.go

```diff
@@ -219,7 +219,7 @@ func createSimulationWallet(cfg *config) error {
 	fmt.Println("Creating the wallet...")
 
 	// Create the wallet database backed by bolt db.
-	db, err := walletdb.Create("bdb", dbPath)
+	db, err := walletdb.Create("bdb", dbPath, true)
```

Comment:
Rather than _always_ set it, this should be read as a new config argument.

Reply:
Do you mean creating new such argument in the application config?